java/client/src/org/openqa/selenium/net/UrlChecker.java

```diff
@@ -79,7 +79,7 @@ public void waitUntilAvailable(long timeout, TimeUnit unit, final URL... urls)
         while (true) {
           for (URL url : urls) {
             try {
-              log.fine("Polling " + url);
+              log.info("Polling " + url);
               connection = connectToUrl(url);
```

Comment:
This is the one mentioned in https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/issues/217